apps/protocol-frontend/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["plugin:@nrwl/nx/react", "../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*", "*.d.ts"],
+  "ignorePatterns": ["!**/*", "*.d.ts", "abnf.ts"],
```

Comment:
Why the ignore on the specific file? Mainly curious -- I saw a lot of warnings were coming from there

Reply:
This is code we copied from siwe. We shouldn't be modifying it and should eventually remove it.